### ISSUE TITLE
Any way to mount to a "real" drive instead of a network drive?

### ISSUE THREAD

**Issue by proviste** [winfsp/sshfs-win#364]
For example to be able to setup a network share

**Comment by oxou** [comment]
Example:
```
sshfs -o "idmap=none,no_readahead,allow_other,reconnect,ServerAliveInterval=15,ServerAliveCountMax=5,noatime,auto 0 0" user@192.168.24.128:/home/user Y:
```

Mounts a physical drive provided by FUSE.
![screenshot](https://user-images.githubusercontent.com/102489121/205484477-a6a9ced7-9176-4a05-b3e4-6eb51fc7b234.png)

**Comment by OskarAtGitHub** [comment]
It would be very useful to be able to mount it as a fake hard disk.
Some programs behave differently if they think it's a network drive. Doing extra rear checks, and strange copy-rename stuff, pretending to be a file system.
This "Local Disk - FUSE" is not enough to convince these programs that it's a local hard disk.
I have no idea what information windows provides programs though, and how much of that can be chosen by the mounting process.

**Comment by fiddyschmitt** [comment]
Yes I've seen programs behave strangely when they detect they're writing to a network share.

Have you tried using a symlink?

https://superuser.com/questions/210824/creating-a-symbolic-link-to-mapped-network-drive-in-windows

Cheers,
Fidel
